allure-pytest/src/allure/utils.py

```diff
@@ -84,8 +84,8 @@ def allure_full_name(nodeid):
 
 
 def step_parameters(func, *a, **kw):
-    all_names = inspect.getargspec(func).args
-    defaults = inspect.getargspec(func).defaults
+    all_names = inspect.getfullargspec(func).args
```

Comment:
yep 
It is does not work correctly for python2.7( But, I think newer python has more priority and it is ok for beta when we have ticket for fix in python2.7. I make release BETA-1.0.1 at friday with some other features or just with that fix. Do you ready to whait a week or have some reasons to get it earler?